tests/cache/tests.py

```diff
@@ -1339,6 +1339,12 @@ def fail_set_multi(mapping, *args, **kwargs):
             failing_keys = cache.set_many({'key': 'value'})
             self.assertEqual(failing_keys, ['key'])
 
+    def test_get_many_accepts_iterator(self):
+        # Regression test for #20584
```

Comment:
No need to include a ticket reference as the test name is self describing.

Reply:
In that case, do you think I should move this test up to `BaseCacheTests` so that it is tested on all Django-official cache backends?

Reply:
I guess it wouldn't hurt to test it against all backends.

Reply:
After looking at the test suite, I think an entirely new unit test for this bug would have made the test suite messier. So I moved the assertion into `BaseCacheTests.test_get_many()`, and kept the comment referencing this regression.